daml.yaml

```diff
@@ -3,8 +3,11 @@
 
 sdk-version: 1.11.1
 name: dabl-integration-core
-version: 1.1.1
+version: 1.2.0
 source: daml/Modules.daml
 dependencies:
 - daml-prim
 - daml-stdlib
+parties:
+- Alice
+- Bob
```

Comment:
Do these actually provide any real value here?

Reply:
Yes. They populate the navigator drop down for parties, which is useful for the local testing I showed last week